CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ In addition to original Keep-a-Changelog, we use following rules:
 - Overview diagram written in asciiflow https://github.com/ricosjp/ruststep/pull/137
 - `Tables` from `DataSection` https://github.com/ricosjp/ruststep/pull/139
 - impl `FromStr` for `Record` and `DataSection` https://github.com/ricosjp/ruststep/pull/140
+- add snapshot files to .gitignore
```

Comment:
```suggestion
- add snapshot files to .gitignore
+ ignore pending snapshot
```

- Please put this line on the top of this section. Upper line means later changes.
- Add link to this pull request